meta-resin-common/recipes-containers/docker/docker/journal.patch

```diff
@@ -0,0 +1,67 @@
+commit 6cdc4ba6cd5178037466c50ebe03a7eb111c43b1
+Author: Nalin Dahyabhai <nalin@redhat.com>
+Date:   Wed Feb 17 15:10:51 2016 -0500
+
+    Try to handle changing names for journal packages
+    
+    When checking if we have the development files for libsystemd's journal
+    APIs, check for either 'libsystemd >= 209' and 'libsystemd-journal'.  If
+    we find 'libsystemd', define the 'journald' tag, which defaults to using
+    the 'libsystemd.pc' file.  If we find the older 'libsystemd-journal',
+    define both the 'journald' and 'journald_compat' tags, which causes the
+    'libsystemd-journal.pc' file to be consulted instead.
+    
+    Signed-off-by: Nalin Dahyabhai <nalin@redhat.com> (github: nalind)
+
```

Comment:
Can you add here a: "Upstream-Status: Backport"

and under it a "Signed-off-by:" to you?